Code/GraphMol/RGroupDecomposition/RGroupCore.cpp

```diff
@@ -260,6 +263,25 @@ std::pair<RWMOL_SPTR, bool> RCore::extractCoreFromMolMatch(
     }
   }
 
+  // Copy over any stereo groups that lie in the extracted core
+  if (!mol.getStereoGroups().empty()) {
+    std::vector<StereoGroup> coreStereoGroups;
+    for (auto &stereoGroup : mol.getStereoGroups()) {
+      std::vector<Atom *> coreStereoAtoms;
+      for (auto stereoGroupAtom : stereoGroup.getAtoms()) {
+        if (auto found = molAtomMap.find(stereoGroupAtom);
+            found != molAtomMap.end()) {
+          coreStereoAtoms.push_back(found->second);
+        }
+      }
+      if (!coreStereoAtoms.empty()) {
+        coreStereoGroups.emplace_back(stereoGroup.getGroupType(),
+                                      coreStereoAtoms);
+      }
+    }
+    extractedCore->setStereoGroups(std::move(coreStereoGroups));
+  }
```

Comment:
Maybe it would make sense to pull this code out into a helper function in `RGroupUtils.h` (or somewhere similar) rather than duplicating it?

Reply:
> Maybe it would make sense to pull this code out into a helper function in `RGroupUtils.h` (or somewhere similar) rather than duplicating it?

I've moved the common code to `RDKit::details::copyStereoGroups` in `ChemTransforms.cpp` (in a similar manner to `updateSubMolConfs`).  I think it is better to have the common code here, rather that `RGroupUtils.h`.  It would be messy to have `ChemTransforms::replaceCore` use RGD code.